DEVELOPING.md

```diff
@@ -135,19 +135,19 @@ def my_command(arg: str) -> None:
     print(f"Argument is {arg}")
 ```
 
-In addition, you need to import this file in the `load_commands` function in `pwndbg/commands/__init__.py`. After this, running `my_command foo` in GDB will print out "Argument is foo".
+In addition, you need to import this file in the `load_commands` function in `pwndbg/commands/__init__.py`. After this, running `my_command foo` in GDB or LLDB will print out "Argument is foo".
 
 # Adding a Configuration Option
 
 ```python
-import pwndbg.gdblib.config
+import pwndbg.config
```

Comment:
Is this correct?
```
In [1]: import pwndbg.config
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 1
----> 1 import pwndbg.config

ModuleNotFoundError: No module named 'pwndbg.config'
```

Reply:
Oh, right, it should be just `import pwndbg`.